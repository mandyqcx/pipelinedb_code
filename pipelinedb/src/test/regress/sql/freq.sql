SELECT freq(freq_agg(x), 10) FROM generate_series(1, 100) AS x;
SELECT freq(freq_agg(x), 10) FROM (SELECT generate_series(1, 100) AS x UNION ALL SELECT generate_series(1, 100) AS x) _;
SELECT freq(freq_agg(x::text), '10') FROM (SELECT generate_series(1, 100) AS x UNION ALL SELECT generate_series(1, 100) AS x) _;
SELECT freq(freq_agg(x::text), 'not here') FROM (SELECT generate_series(1, 100) AS x UNION ALL SELECT generate_series(1, 100) AS x) _;
SELECT freq(freq_agg(x::text), 'first') FROM (SELECT 'first' AS x FROM generate_series(1, 100) UNION ALL SELECT 'second' AS x FROM generate_series(1, 100)) _;

SELECT freq(freq_merge_agg, 50)
  FROM (SELECT freq_merge_agg(freq_agg) FROM
    (SELECT freq_agg(x) FROM generate_series(1, 100) AS x
      UNION ALL
     SELECT freq_agg(x) FROM generate_series(1, 100) AS x) _) _;