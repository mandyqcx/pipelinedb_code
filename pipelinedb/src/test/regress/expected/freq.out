SELECT freq(freq_agg(x), 10) FROM generate_series(1, 100) AS x;
 freq 
------
    1
(1 row)

SELECT freq(freq_agg(x), 10) FROM (SELECT generate_series(1, 100) AS x UNION ALL SELECT generate_series(1, 100) AS x) _;
 freq 
------
    2
(1 row)

SELECT freq(freq_agg(x::text), '10') FROM (SELECT generate_series(1, 100) AS x UNION ALL SELECT generate_series(1, 100) AS x) _;
 freq 
------
    2
(1 row)

SELECT freq(freq_agg(x::text), 'not here') FROM (SELECT generate_series(1, 100) AS x UNION ALL SELECT generate_series(1, 100) AS x) _;
 freq 
------
    0
(1 row)

SELECT freq(freq_agg(x::text), 'first') FROM (SELECT 'first' AS x FROM generate_series(1, 100) UNION ALL SELECT 'second' AS x FROM generate_series(1, 100)) _;
 freq 
------
  100
(1 row)

SELECT freq(freq_merge_agg, 50)
  FROM (SELECT freq_merge_agg(freq_agg) FROM
    (SELECT freq_agg(x) FROM generate_series(1, 100) AS x
      UNION ALL
     SELECT freq_agg(x) FROM generate_series(1, 100) AS x) _) _;
 freq 
------
    2
(1 row)

